pkg/kopia/command/snapshot.go

```diff
@@ -18,24 +18,25 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/kanisterio/kanister/pkg/kopia"
 	"github.com/kanisterio/kanister/pkg/utils"
 )
 
 const (
-	requireLogLevelInfo = true
+	requireLogLevelInfo        = true
+	ManifestTypeSnapshotFilter = "type:snapshot"
```

Comment:
No need to export the const if not being used outside this package.
```suggestion
	manifestTypeSnapshotFilter = "type:snapshot"
```

Reply:
Done !